src/service/vaults.ts

```diff
@@ -308,7 +306,11 @@ export const watchVaultFactory = () => {
       [Kind.Data, path],
       value => {
         console.debug('got update', path, value);
-        useVaultStore.getState().setVaultMetrics(id, value);
+        useVaultStore.getState().setVaultMetrics(id, {
+          ...value,
+          // Invert the lockedQuote to be congruent with oracle price quotes.
+          lockedQuote: value.lockedQuote && invertRatio(value.lockedQuote),
```

Comment:
are there distinct names that would make sense in this codebase? for when someone is reading the symbol out of context.

Reply:
I'm not sure if I can think of a better name, it might be easiest to just keep the on-chain name here.